app/models/post.rb

```diff
@@ -0,0 +1,24 @@
+class Post < ApplicationRecord
+  # Associations
+  belongs_to :author, class_name: 'User'
+  has_many :comments
+  has_many :likes
+
+  # Attributes
+  attribute :title, :string
+  attribute :text, :text
+  attribute :comments_counter, :integer, default: 0
+  attribute :likes_counter, :integer, default: 0
+
+  # Callback
+  after_save :update_user_posts_counter
+
+  # Methods
+  def update_user_posts_counter
+    author.update(posts_counter: author.posts_counter + 1)
+  end
+
```

Comment:
- [ ] Kindly take note that I got the error message below in the process of checking your implementation. If you follow the previous reviewer suggestion, I think this method will work as expected. 👍 

```suggestion
  # Methods
  def update_user_posts_counter
    author.update(posts_counter: author.posts.count + 1)
  end
```

<img width="569" alt="Untitled" src="https://github.com/VaheStepanyan100/blog-app/assets/103386183/198f0c10-843c-4d51-9a9a-b1a0710a6a46">

Reply:
well resolved! ✔️